### PR TITLE
Bring slack logo back to the Footer

### PR DESCRIPTION
**Pull request by mshaaban088**
Reverts storybooks/storybook#2819

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

File: docs/src/pages/_users.yml [hunk 1]
```diff
@@ -5,12 +5,12 @@ airbnb:
...
-#   logo: slack.svg
-#   title: Slack Emoji Picker
-#   description: Slack improves performance and maintainability with Storybook!
-#   demo: https://slack.engineering/rebuilding-slacks-emoji-picker-in-react-bfbd8ce6fbfe
-#   site: http://slack.com
+slack:
+  logo: slack.svg
+  title: Slack Emoji Picker
+  description: Slack improves performance and maintainability with Storybook!
+  demo: https://slack.engineering/rebuilding-slacks-emoji-picker-in-react-bfbd8ce6fbfe
```
Comment (danielduan) [inline docs/src/pages/_users.yml]: wait, I'm confused. this is not a demo.

Comment (mshaaban088) [reply docs/src/pages/_users.yml]: This is not a demo per se, but they demonstrated how storybook helped them developing UI development environment. It doesn't qualify as a site either from my understanding, or am I missing something?

Comment (danielduan) [reply docs/src/pages/_users.yml]: Demo to me means an actual instance of storybook that a user can play with and see how someone else used it. The rest of the demo links are all usage examples.

The medium article is more of a case study and talks very little about the specifics of their storybook implementation.

I went in a while back and sorted the list to prioritize good demos at the top. This should would probably be at the bottom.

Comment (mshaaban088) [reply docs/src/pages/_users.yml]: Fair enough!

We should remove that link then, and bring it back once they share it. WDYT?

**Pull request merged**